lib/src/config/route/routes.dart

```diff
@@ -0,0 +1,8 @@
+abstract class Routes {
+  static const String homeRouteName = "/home";
```

Comment:
By definition you always need to have a home route that is '/', ideally the splash screen take that route, you can replace '/home' with '/', but remember to change on the next module when you define the splash screen

Reply:
Okay, anyway i've changed it to `/` and in the next module i implement the correctly `/` with the slpash screen and `/home` for the home page